### PR TITLE
Issue 4

### LINKED ISSUE #4: Building a host frame

### PR DESCRIPTION
**Pull request by tylerbutler**
Fixes #4.

See the output of this pull request at http://wopi.tylerbutler.com/issue_4/

### CODE REVIEW

File: docs/docs_source/hostpage.rst [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+are of the form ``wd*``. When you receive these query string parameters on your host page URLs, you must pass them,
+unchanged, to the Office Online iframe.
+
+In addition, if the `replaceState`_ method from the HTML5 History API is available in the user's browser, you should
+remove the following parameters from your host page URL after passing them to the Office Online iframe:
+
+* wdPreviousSession
+* wdPreviousCorrelation
+
+Other ``wd*`` parameters do not need to be removed from the host page URL.
```
Comment (MarcinJuraszek) [inline docs/docs_source/hostpage.rst]: `do not need` is too weak. Should be `must not`

File: samples/SampleHostPage.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            left: 0;
+            right: 0;
+            bottom: 0;
+            margin: 0;
+            border: none;
+            display: block;
+        }
+    </style>
+</head>
+<body style="margin: 0; padding: 0; overflow:hidden;">
```
Comment (MarcinJuraszek) [inline samples/SampleHostPage.html]: Should be css instead of inline styling.

Comment (tylerbutler) [inline samples/SampleHostPage.html]: I was trying to keep the sample to a single file. You think a comment in the HTML pointing out that in a real page the styles should be separate is enough, or should I just create a CSS file and move the styles there?

File: docs/docs_source/hostpage.rst (hunk 1 above)
Comment (tylerbutler) [inline docs/docs_source/hostpage.rst]: Will change.

File: docs/docs_source/hostpage.rst [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+sections below will refer to it in more detail.
+
+
+..  _Passing access tokens securely:
+
+Passing access tokens securely
+------------------------------
+
+It is important, for security purposes, that access tokens not be passed to the Office Online iframe as a query
+string parameter. Doing so would greatly increase the likelyhood of token leakage. To avoid this problem, hosts
```
Comment (tylerbutler) [inline docs/docs_source/hostpage.rst]: Spelling error...

File: samples/SampleHostPage.html (hunk 2 above)
Comment (MarcinJuraszek) [inline samples/SampleHostPage.html]: No. You can keep it in on e file, just move it where `#office_form` is defined.

Comment (tylerbutler) [inline samples/SampleHostPage.html]: Ahhh, I misunderstood what you were referring too. Got it; I'll update this.

File: docs/docs_source/hostpage.rst [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+..  _wd Parameters:
+
+Passing through wd* parameters
+------------------------------
+
+Office Online will sometimes pass additional query string parameters to your host page. These query string parameters
+are of the form ``wd*``. When you receive these query string parameters on your host page URLs, you must pass them,
+unchanged, to the Office Online iframe.
+
+In addition, if the `replaceState`_ method from the HTML5 History API is available in the user's browser, you should
```
Comment (KnightSky) [inline docs/docs_source/hostpage.rst]: There's an _  after replaceState. Not really sure it's purpose.

Comment (tylerbutler) [inline docs/docs_source/hostpage.rst]: That's an [external link](http://sphinx-doc.org/rest.html#external-links) in reStructuredText. :) The link target is at the bottom of the file on line 96.

**Comment by KnightSky** [comment]
The changes look good. Not exactly sure how to 'sign off' other than to merge the pull request but I'll leave that to you.

**Pull request merged**